src/IO/InterfaceImpl.cpp

```diff
@@ -151,6 +114,30 @@ void InterfaceImpl::calibrateTelemetry()
 
 std::shared_ptr<sensorsData> InterfaceImpl::getLatest()
 {
+#if USE_LOGGER
```

Comment:
Does it make sense to put this in `updateInputs` instead?

Reply:
Yeah, I think it would make a bit more sense to put it in `updateInputs`. @ZachLegros, was there a specific reason to put it there?